plugins/textpattern.md

```diff
@@ -53,7 +53,12 @@ There are three types of patterns: inline, block, and replacement patterns. Inli
 
 #### Inline patterns
 
-Inline patterns have a `start` and an `end`, and either a `format` or a `cmd` and optional `value` property. This allows for patterns to be used to either apply a format or execute a command, optionally with the given value.
+Inline patterns must have the following:
+* A `start` and an `end`
+* A `format` or a `cmd` 
+* An optional `value` property. 
```

Comment:
The `value` property is only allowed when `cmd` is specified.

Reply:
Fixed